### PR TITLE
refactor: merge nodejs tests into one pipeline

### PR DESCRIPTION
**Pull request by Mini256**
This pull request merges two containers for a TypeORM test. The previous default container was "golang," which has been replaced by "nodejs."

### CODE REVIEW

File: pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_typeorm_test.groovy [hunk 1]
```diff
@@ -117,28 +110,26 @@ pipeline {
...
+                                            bash ${WORKSPACE}/scripts/PingCAP-QE/tidb-test/start_tikv.sh
+                                            export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                            export TIKV_PATH="127.0.0.1:2379"
+                                            export TIDB_TEST_STORE_NAME="tikv"
+                                            cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                        else
+                                            export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                            export TIDB_TEST_STORE_NAME="unistore"
+                                            cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                        fi
```
Comment (wuhuizuo) [inline pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_typeorm_test.groovy:131]:
```suggestion
                                        export TIDB_SERVER_PATH="\$(pwd)/bin/tidb-server"
                                        export TIDB_TEST_STORE_NAME="${TEST_STORE}"
                                        if [[ "${TEST_STORE}" == "tikv" ]]; then
                                            echo '[storage]\nreserve-space = "0MB"'> tikv_config.toml
                                            bash ${WORKSPACE}/scripts/PingCAP-QE/tidb-test/start_tikv.sh
                                            export TIKV_PATH="127.0.0.1:2379"
                                        fi

                                       cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}

```

File: pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_nodejs_test.groovy [hunk 2]
```diff
@@ -78,25 +78,25 @@ pipeline {
...
             }
         }
-        stage('TypeORM Tests') {
+        stage('Node.js Tests') {
             matrix {
                 axes {
                     axis {
                         name 'TEST_PARAMS'
-                        values 'typeorm_test ./test.sh'
+                        values 'prisma_test ./test.sh', 'typeorm_test ./test.sh', 'sequelize_test ./test.sh'
```
Comment (purelind) [inline pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_nodejs_test.groovy:86]: Suggest making a modification, directly adding the execution cmds `cd prisma_test && chmod +x *.sh && ./test.sh`

Comment (Mini256) [reply pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_nodejs_test.groovy:86]: Great. More simple

**Pull request merged**